src/frontend/src/utils/status.ts

```diff
@@ -1,7 +1,6 @@
 import { IPiaForm } from '../types/interfaces/pia-form.interface';
 import { BannerText } from '../pages/PIAForm/BannerStatus/messages';
-import { PiaStatuses } from '../constant/constant';
-import { SubmitButtonTextEnum } from '../pages/PIAForm';
+import { PiaStatuses, SubmitButtonTextEnum } from '../constant/constant';
```

Comment:
Are you using PiaStatuses?

Reply:
Not sure. It was greyed out so I believe it was not used. Cleaning up some of the lint warnings.